### ISSUE TITLE
Make decisions on currently unused rules

### ISSUE THREAD

**Issue by lydell** [keithamus/eslint-config-strict#34]
```
$ ./node_modules/.bin/eslint-find-rules es6.js --unused

unused rules

capitalized-comments                       class-methods-use-this
func-name-matching                         max-lines
max-statements-per-line                    no-await-in-loop
no-multi-assign                            no-native-reassign
no-negated-in-lhs                          no-prototype-builtins
no-restricted-properties                   no-return-await
no-spaced-func                             no-unsafe-negation
no-useless-escape                          no-useless-return
prefer-destructuring                       prefer-promise-reject-errors
require-await                              sort-keys
unicode-bom
```

**Comment by keithamus** [comment]
My opinions:

 - ❌ `capitalized-comments` - Enforcing commenting is tricky as they're used lots for , I'd rather avoid doing it.
 - ✅ `class-methods-use-this` - Yes, if it doesn't use this is shouldn't be in class. Only in es6.js though
 - ✅ `func-name-matching` - Seems sensible
 - ❌ `max-lines` - Seems like a convention that would just be disabled once the max lines is reached. Separation of files feels more nuanced than line count.
 - ✅ `max-statements-per-line` - Feels like a sensible rule, not sure on statement count. Maybe 5?
 - 🤷‍♂️ `no-await-in-loop` - Not sure, `Promise.all` is not _always_ wanted.
 - ✅ `no-multi-assign` - Yes, can cause readability issues
 - ❌ `no-native-reassign` - deprecated over [no-global-assign](https://github.com/keithamus/eslint-config-strict/blob/b5ef205b426ce1809f73ee704e2a04c49051de51/es5.js#L151). We could zero-out and add a comment?
 - ❌ `no-negated-in-lhs` - deprecated
 - ✅ `no-prototype-builtins` - yup, writing defensible code is good, this encourages that
 - ✅ `no-restricted-properties` - could be good for `__proto__`, `__defineGetter__` etc.
 - ✅ `no-return-await` - yup, as description says its probably misuse of function
 - ❌ `no-spaced-func` - deprecated for [func call spacing](https://github.com/keithamus/eslint-config-strict/blob/b5ef205b426ce1809f73ee704e2a04c49051de51/es5.js#L395)
 - ✅ `no-unsafe-negation`
 - ✅ `no-useless-escape`
 - ✅ `no-useless-return`
 - ✅ `prefer-destructuring` - yup, aids readability - only put in es6.js though
 - ✅ `prefer-promise-reject-errors` - yup, Promises.reject should always take error objects
 - ✅ `require-await`
 - ❌ `sort-keys` - found with imports, there are stronger semantics than alphabetical order, too burdensome
 - 🤷‍♂️ `unicode-bom` - I have no preference, don't understand the use case. Erring on no though.

**Comment by keithamus** [comment]
@lydell I'd like to hear your opinions on these!

**Comment by lydell** [comment]
I more or less agree with you on all of them!

- I don't think we should enable the two rules that you were unsure of.
- `max-statements-per-line`: I'd go with the ESLint default value of max 1 here.

Would you like a PR for this, or do you prefer implementing it yourself?

**Comment by keithamus** [comment]
Okay excellent. Happy to have `max-statements-per-line` be 1. 

Happy for you to make the PR if you'd like!

**Comment by lydell** [comment]
Fixed by #35.
